app/controlplane/pkg/biz/apitoken_integration_test.go

```diff
@@ -374,3 +374,45 @@ func (s *apiTokenTestSuite) SetupTest() {
 	s.t5, err = s.APIToken.Create(ctx, randomName(), nil, nil, s.org.ID, biz.APITokenWithProject(s.p1))
 	require.NoError(s.T(), err)
 }
+
+func (s *apiTokenTestSuite) TestUpdateLastUsedAt() {
+	ctx := context.Background()
+
+	s.Run("update last used at", func() {
+		token, err := s.APIToken.Create(ctx, randomName(), nil, nil, s.org.ID)
+		require.NoError(s.T(), err)
+		assert.Nil(s.T(), token.LastUsedAt)
```

Comment:
since you  are using a test suite you can rename all these `assert.XYZ(s.T(), ...)` for `s.XYZ`, for example `		assert.Nil(s.T(), token.LastUsedAt)` => 		s.Nil(token.LastUsedAt)`